src/cron/updateSourcePublicThreshold.ts

```diff
@@ -16,24 +17,43 @@ export const updateSourcePublicThreshold: Cron = {
       .update()
       .set({ flags: updateFlagsStatement({ publicThreshold: true }) })
       .where(
-        `
-          "type" = 'squad' AND
-          "image" IS NOT NULL AND
-          "image" != '${SQUAD_IMAGE_PLACEHOLDER}' AND
-          "description" IS NOT NULL AND
-          (flags->>'publicThreshold')::boolean IS NOT TRUE AND
-          (flags->>'vordr')::boolean IS NOT TRUE AND
-          (
-          ((flags->>'totalMembers')::int >= 3 AND (flags->>'totalPosts')::int >= 3) OR
-          exists (select 1
-            from "content_preference" cp
-            join "user" u on cp."userId" = u.id
-            where cp."referenceId" = "source".id and cp.type = 'source'
-                and cp.flags->>'role' = 'admin' and u.reputation >= :threshold
-                and (u.flags->>'vordr')::boolean IS NOT TRUE
-          ))
+        /* sql */ `
+          "type" IN (:...sourceTypes)
+          AND (
+            "type" != :squadSourceType
+            OR (
+              "type" = :squadSourceType
+              AND "image" IS NOT NULL
+              AND "image" != :imagePlaceholder
+              AND "description" IS NOT NULL
+            )
+          )
+          AND (flags->>'publicThreshold')::boolean IS NOT TRUE
+          AND (flags->>'vordr')::boolean IS NOT TRUE
```

Comment:
Can you explain what changes you did for reference?

Reply:
Ah, yeah, all lines changed since I made it more readable 🙈 

Changed the first check for type to be `IN` instead of `=` so that it checks for squad and user source.

I wrapped the `image` and `description` to only be checked when source type is `squad` (or `:squadSourceType`).

Then I also wrapped the `flags.totalMembers` and `flags.totalPosts` to only be checked when source type is `squad`

Reply:
Can check the commit 9f1aba35e4867c059d9419a47be0a5ec6f8ce5ed, which is where I did the change for checks.